Libraries/Blob/FileReader.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule FileReader
+ * @flow
+ */
+
+'use strict';
+
+const EventTarget = require('event-target-shim');
+const Blob = require('Blob');
+const { FileReaderModule } = require('NativeModules');
+
+type ReadyState =
```

Comment:
Using `Blob`/`BlobTypes` analogy, should we move types to `FileReaderTypes`?

Reply:
`BlobTypes` is imported in multiple places, so Improved to a separate file. Probably not needed here.